Modules/Tests/YosemiteTests/Stores/SystemPlugin/SystemPluginsUpsertUseCaseTests.swift

```diff
@@ -0,0 +1,241 @@
+import Foundation
+import Testing
+
+import Fakes
+import Storage
+@testable import Networking
+@testable import Yosemite
+
+@MainActor
+struct SystemPluginsUpsertUseCaseTests {
+    private let siteID: Int64 = 10
+    private var storageManager: MockStorageManager!
+    private var viewStorage: StorageType {
+        storageManager.viewStorage
+    }
+
+    init() async throws {
+        storageManager = MockStorageManager()
+        storageManager.reset()
+    }
+
+    @Test func upsert_inserts_plugins_in_storage() async throws {
+        // Given
+        let activePlugin = SystemPlugin.fake().copy(siteID: siteID, plugin: "woocommerce/woocommerce.php", name: "WooCommerce")
```

Comment:
If we don't need names anymore on these we could update all tests/cases to get rid of the property:

```suggestion
        let activePlugin = SystemPlugin.fake().copy(siteID: siteID, plugin: "woocommerce/woocommerce.php")
```

Reply:
Yea makes sense, the plugin name removed in https://github.com/woocommerce/woocommerce-ios/pull/15915/commits/4dde32e96fb3c37077717bd9b3dacc9c3a541b0d.